_config.yml

```diff
@@ -0,0 +1,10 @@
+theme: jekyll-theme-cayman
+data_dir: ./yml
+layouts_dir: ./browser/_layouts
+exclude:
+  - Gemfile
+  - /deprecated
```

Comment:
Since this is still available in git history, can it just be removed?  (Or moved somewhere else.)

Reply:
Not sparking joy?